test/modularize_post_js.js

```diff
@@ -0,0 +1,10 @@
+#preprocess
+
+// This file get included via `--extern-post-js` in order to run tests
+// that are built with `-sMODULARIZE`.
+
+#if PTHREADS
+// Avoid instantiating the module on pthreads.
+if (!isPthread)
+#endif
+{{{ EXPORT_NAME }}}();
```

Comment:
This seems like a general thing that all users will want, but I don't have a good idea for how to provide that. We could make `Module()` do nothing at all on pthreads, in theory, but that seems risky, even if I can't think of a use for it.

Reply:
Yes, I thank that might make sense, but doesn't have to be part of this change.  We can consider that as followup.